examples/topics/stats/density.py

```diff
@@ -4,7 +4,7 @@
 .. bokeh-example-metadata::
     :sampledata: cows
     :apis: bokeh.plotting.figure.varea
-    :refs: :ref:`ug_topics_stats_density`
+    :refs: :ref:`ug_topics_stats_sinaplot`
```

Comment:
I know this has already been merged, but I just want a bit of clarification here. Shouldn't the `ref` point to the KDE sub-topic rather than the sinaplot one below it?

Reply:
```suggestion
    :refs: :ref:`ug_topics_stats_kde`
```

Reply:
@Azaya89 Sorry for the late answer. I think it is fine because the `Kernel density estimation` is right above the `SinaPlot` section and I think can easily find all information. If you think it is better to change these for a better understanding, feel free to open a PR. But please keep in mind, that the ref `ug_topics_stats_sinaplot` isn't needed anymore and should be removed in `bokeh/docs/bokeh/source/docs/user_guide/topics/stats.rst:L46`, too.